base/uk.ac.stfc.isis.ibex.ui.motor/src/uk/ac/stfc/isis/ibex/ui/motor/views/MotorInfoAdvView.java

```diff
@@ -0,0 +1,220 @@
+	
+	private static final Image ENERGISED_ENABLED = ResourceManager.getPluginImage("uk.ac.stfc.isis.ibex.ui.motor", "icons/power_on.png");
+	private static final Image ENERGISED_DISABLED = ResourceManager.getPluginImage("uk.ac.stfc.isis.ibex.ui.motor", "icons/power_off.png");
+
+	public MotorInfoAdvView(Composite parent, int style, MinimalMotorViewModel minimalMotorViewModel) {
+		super(parent, style, minimalMotorViewModel); 
+		
+        GridLayout glMotorComposite = new GridLayout(1, false);
+		glMotorComposite.verticalSpacing = MOTOR_COMPOSITE_VERTIAL_SPACING;
+		glMotorComposite.marginWidth = MOTOR_COMPOSITE_MARGIN_WIDTH;
+		glMotorComposite.marginHeight = MOTOR_COMPOSITE_MARGIN_HEIGHT;
+        setLayout(glMotorComposite);
+   
+        motorName = new Label(this, SWT.NONE);
+		motorName.setAlignment(SWT.CENTER);
+		GridData gdMotorName = new GridData(SWT.TOP, SWT.TOP, false, false);
+		gdMotorName.minimumWidth = MOTOR_NAME_MINIMUM_WIDTH;
+		gdMotorName.widthHint = MOTOR_NAME_WIDTHHINT;
+		motorName.setLayoutData(gdMotorName);
+		motorName.setText("Motor name");
+		
+		createSetpointAndReadback();
+        
+        createLimits();
+        
+        createErrorAndOffset();
+        
+        iconContainer = new Composite(this, SWT.NONE);
+        
+        iconContainer.setLayoutData(new GridData(SWT.FILL, SWT.TOP, false, false));
+        
+        GridLayout iconContainerLayout = new GridLayout(3, false);
+        iconContainerLayout.marginHeight = 0;
+        iconContainerLayout.marginWidth = 0;
+        iconContainerLayout.horizontalSpacing = 2;
+        iconContainerLayout.verticalSpacing = 0;
+        iconContainer.setLayout(iconContainerLayout);
+		
+        minimalMotionIndicator = new MinimalMotionIndicator(iconContainer, SWT.NONE);
+		minimalMotionIndicator.setLayoutData(new GridData(SWT.FILL, SWT.TOP, false, false));
+        minimalMotionIndicator.setMotor(this.getViewModel().getMotor());
+        
+        encoderStatus = new BooleanImageLabel(iconContainer, ENCODER_ENABLED, ENCODER_DISABLED, "Encoder present", "Encoder not present");
+        encoderStatus.enable(minimalMotorViewModel.getUsingEncoder());
+        minimalMotorViewModel.addUiThreadPropertyChangeListener("usingEncoder", 
+        		evt -> encoderStatus.enable(minimalMotorViewModel.getUsingEncoder()));
+        
+        energisedStatus = new BooleanImageLabel(iconContainer, ENERGISED_ENABLED, ENERGISED_DISABLED, "Motor energised", "Motor not energised");
+        energisedStatus.enable(minimalMotorViewModel.getEnergised());
+        minimalMotorViewModel.addUiThreadPropertyChangeListener("energised", 
+        		evt -> energisedStatus.enable(minimalMotorViewModel.getEnergised()));
+        
+        setMouseListeners();
+        
+        bind();
+	}
+	
+	private Label createMotorValueLabel(final Composite container) {
+		final var label = new Label(container, SWT.NONE);
```

Comment:
Can we add a tooltip to this too please? It does not need to be dynamic just a description of the field